.github/workflows/ci.yml

```diff
@@ -5,18 +5,19 @@ on:
     branches: [main, release/*]
   pull_request:
     branches: [main, release/*]
-    paths: [nexus/**, protos/**]
+    paths: [nexus/**, protos/**, .github/workflows/ci.yml]
 
 jobs:
   build:
     strategy:
       matrix:
         runner: [ubicloud-standard-2-ubuntu-2204-arm]
+        postgres-version: [14, 15, 16, 17]
```

Comment:
can add PG 13 here as well; 12 will be EOL soon

Reply:
we have customers from pg 13?

Reply:
we allow PG13 peers to be created